developer/src/kmldmlc/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": "../../../tsconfig-base.json",
+
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es2017",
+    "moduleResolution": "node",
+    "sourceMap": true,
+    "outDir": "build/",
+    "rootDir": "src/",
+    "declaration": true,
+    "alwaysStrict": true,
+    "noImplicitThis": true,
+    "noImplicitReturns": true,
+    "noImplicitAny": true,
+    "strictBindCallApply": true,
+    "strictFunctionTypes": true,
+    "noUnusedLocals": true
+  },
+  "include": [
+    "src/**/*.ts"
+  ],
+  "references": [
+    { "path": "../../../common/web/keyman-version" },
```

Comment:
and keyboardprocessor_ldml.ts too, soon?

Reply:
I'll add it once we I it :)